CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+
+# Contributing to the paper (or papers) about the R course
+
+Adding content to the repository:
+
+1. Have your own fork or create a branch from the master `paper-rcourse` repo.
+2. Work on a defined issue that was assigned to you or that you self-assigned.
+3. Submit a PR specific to the issue to close it. If you are still working on it,
```

Comment:
Maybe add that if you write "closes #"  in the PR comment it will auto-close the issue?
`Write "Closes #issue_number" in the comment of your pull request if the PR closes the issue.`